google/cloud/storage/blob.py

```diff
@@ -37,14 +37,14 @@
 import re
 import warnings
 
-import six
-from six.moves.urllib.parse import parse_qsl
+import six  # type: ignore
+from six.moves.urllib.parse import parse_qsl  # type: ignore
 from six.moves.urllib.parse import quote
 from six.moves.urllib.parse import urlencode
 from six.moves.urllib.parse import urlsplit
 from six.moves.urllib.parse import urlunsplit
 
-from google import resumable_media
+from google import resumable_media  # type: ignore
```

Comment:
I'm adding typing support for `resumable_media` in this PR: https://github.com/googleapis/google-resumable-media-python/pull/287.